politeiad/api/v1/v1.go

```diff
@@ -245,6 +250,7 @@ type NewRecordReply struct {
 type GetUnvetted struct {
 	Challenge string `json:"challenge"` // Random challenge
 	Token     string `json:"token"`     // Censorship token
+	Version   string `json:"version"`   // Record version
```

Comment:
Why is this a string?

Reply:
Its a string because the version is a string in gitbe.  These v1 changes were left over from when the tstorbe still used the v1 API.  I've reverted all `politeiad/api/v1` and `politeiad/backend` changes since the tstorebe uses the `v2` api and the `backendv2` interface.